server/src/main/java/io/druid/initialization/Initialization.java

```diff
@@ -118,33 +119,46 @@
   /**
    * Used for testing only
    */
-  protected static void clearLoadedModules()
+  static void clearLoadedModules()
   {
     extensionsMap.clear();
   }
 
+  /**
+   * Used for testing only
+   */
+  static Map<String, URLClassLoader> getLoadersMap()
```

Comment:
It has to be package protected, for the test class to be able to access it, so this is fine.